book/07-git-tools/sections/reset.asc

```diff
@@ -71,7 +71,7 @@ The index is not technically a tree structure -- it's actually implemented as a
 
 ===== The Working Directory
 
-Finally, you have your working directory.
+Finally, you have your working directory (also referred to in much of the literature as the ``working tree'').
```

Comment:
Seems a bit heavy. Maybe `sometimes referred to as the ``working tree''`?

Reply:
In Asciidoc, you have to opt in to curly quotes, and their direction matters. Check [the cheatsheet](https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/) and scroll down to "Curved Quotation Marks".